### PR TITLE
Add Python 3.10 to test matrix.

### PR DESCRIPTION
**Pull request by hramezani**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have run the `pre-commit run` command to format and lint.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] I have added my name and/or github handle to `AUTHORS.rst`
- [ ] I have added my change to `CHANGES.rst`
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ This app supports the following combinations of Django and Python:
 ==========  =======================
 2.2         3.6, 3.7, 3.8, 3.9
 3.1         3.6, 3.7, 3.8, 3.9
-3.2         3.6, 3.7, 3.8, 3.9
+3.2         3.6, 3.7, 3.8, 3.9, 3.10
 ==========  =======================
```
Comment (jezdez) [inline README.rst:47]:
```suggestion
==========  ========================
```

File: README.rst [hunk 2]
```diff
@@ -43,8 +43,8 @@ This app supports the following combinations of Django and Python:
...
```
Comment (jezdez) [inline README.rst:43]:
```suggestion
==========  ========================
```

**Review by jezdez — APPROVED** [review]
Thank you!

**Comment by hramezani** [comment]
Thanks @jezdez for the review

**Pull request merged**